README.md

```diff
@@ -21,6 +21,9 @@ Both examples have the option of preparing the default biokepi-work directory
 with
 [`b37decoy_20160927.tgz`](https://storage.googleapis.com/hammerlab-biokepi-data/precomputed/b37decoy_20160927.tgz).
 
+For more “administrator” roles, we also provide `secotrec-make-dockerfiles`, as
```

Comment:
I'd maybe rephrase to something like "For those administrating Ketrew deployments, we also…"